src/vita-export-parse.c

```diff
@@ -55,11 +55,26 @@ int process_functions(yaml_node *entry, vita_library_export *export) {
 		vita_export_symbol *symbol = malloc(sizeof(vita_export_symbol));
 		symbol->name = strdup(key->value);
 
-		// Hacky logic to protect against nid conflicts.
-		char *func_name_for_nid = malloc(key->len + 11);
-		int outlen = snprintf(func_name_for_nid, key->len + 10, "%s_%08X", key->value, exported_func_count++);
-		symbol->nid = sha256_32_vector(1, (uint8_t **)&func_name_for_nid, (size_t *)&outlen);
-		free(func_name_for_nid);
+		if(export->version == 0 || export->version == 1 || export->syscall != 0){
+			symbol->nid = sha256_32_vector(1, (uint8_t **)&key->value, (size_t *)&key->len);
+		}else{
```

Comment:
minor thing, add space it
```
if (....) {
}
else {
}
```
like line 221~274

please change same thing of line 143~145

Reply:
Done